src/qtism/data/storage/xml/marshalling/OutcomeDeclarationMarshaller.php

```diff
@@ -115,6 +119,11 @@ protected function unmarshall(DOMElement $element)
             $object->setCardinality($baseComponent->getCardinality());
             $object->setDefaultValue($baseComponent->getDefaultValue());
 
+            // Set external scored attribute
+            if ($element->hasAttribute('externalScored')) {
```

Comment:
Here already `externalScored` :) How could you make it more consistent?

Reply:
+1